python/lsst/meas/base/forcedPhotImage.py

```diff
@@ -143,6 +140,23 @@ def runDataRef(self, dataRef, psfCache=None):
         if psfCache is not None:
             exposure.getPsf().setCacheSize(psfCache)
         refCat = self.fetchReferences(dataRef, exposure)
+
+        measCat = self.run(exposure, refCat, refWcs, dataRef)
+
+        self.writeOutput(dataRef, measCat)
+
+    def run(self, exposure, refCat, refWcs, dataRef):
+        """!Measure a single exposure with forced detection for a reference catalog.
+
+        @param[in]  exposure  The measurement image upon which to perform forced detection.
+        @param[in]  refCat    The reference catalog of sources to measure.
+        @param[in]  refWcs    The WCS for the references.
+        @param[in]  dataRef   An lsst.daf.persistence.ButlerDataRef.  Needed for method calls and logging.
+
+        @param[out] measCat   Reference sources are generated with generateMeasCat() in the measurement
```

Comment:
This is a returned value instead of an argument that is modified, so use `@return` instead of `param[out]`.

Also I find the description confusing. I assume `measCat` is measured sources, not reference sources, but the description starts with "Reference sources". Consider something like "Measured sources" or "Measured forced sources" or "Sources measured by generateMeasCat"